browser/src/react-redux/index.jsx

```diff
@@ -1,16 +1,20 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import {Route, BrowserRouter} from 'react-router-dom';
-import {Provider} from 'react-redux';
-import store from './store';
+import React from "react";
+import ReactDOM from "react-dom";
+import { Route, BrowserRouter } from "react-router-dom";
+import { Provider } from "react-redux";
+import store from "./store";
 
-import App from './App/App.react.jsx';
+import App from "./App/App.react.jsx";
+import SignupForm from "./App/Signup/SignupForm";
 
 ReactDOM.render(
-    <Provider store={store}>
-        <BrowserRouter>
-            <Route path='/' component={App} />
-        </BrowserRouter>
-    </Provider>,
-    document.getElementById('app')
-);
\ No newline at end of file
+  <Provider store={store}>
+    <BrowserRouter>
+      <div>
+        <Route exact path="/" component={App} />
+        <Route path="/signup" component={SignupForm} />
```

Comment:
We probably want to render this in the App component and keep App is the highest level. That way, App can contain useful global context. Conditionally render the signup form route inside of App.